Wiley Online Library.js

```diff
@@ -255,6 +255,11 @@ function scrapeBibTeX(doc, url, pdfUrl) {
 				item.title = ZU.capitalizeTitle(item.title, true);
 			}
 			
+			//date in the cochraine library RIS is wrong
+			if (ZU.xpathText(doc, '//meta[@name="citation_book_title"]/@content') == "The Cochrane Library") {
```

Comment:
Hmmm - I was thinking testing is more future-proof, but I can see how that could be a matter of taste. Also, we may not ever get there.